### DISCUSSION TITLE
%E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/1130721%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1

### DISCUSSION THREAD

**Discussion by giscus** [cit-hall-43/cit-hall-43.github.io#3] [Q&A]
# %E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/1130721%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1

讚美主

https://cit-hall-43.github.io/%E4%B8%BB%E6%97%A5%E8%B3%87%E6%96%99/1130721%E4%B8%BB%E6%97%A5%E7%B0%A1%E5%A0%B1.html

**Comment by cit-hall-43** [comment]
讚美主新增詩歌176

**Reply by cit-hall-43** [reply]
最後新增詩歌補充本0712

**Comment by cit-hall-43** [comment]
神所給予我們的終極目標就是進入美地，但我們仍需要捨去自己並為主爭戰才能進入
